libraries/chain/block_log.cpp

```diff
@@ -1392,7 +1392,7 @@ namespace sysio { namespace chain {
       if (!first_block_file.empty()) {
          return block_log_data(first_block_file).get_preamble().chain_context;
       }
-      
+
```

Comment:
Add this back in so there are no differences, which will make my merge of the templated block_log much easier.

Reply:
i think git can handle merging the removal of a line full of spaces.  Reverting this change (without completely force-pushing a new merge history) would be counter-productive.